descwl_shear_sims/psfs/dmpsfs.py

```diff
@@ -78,14 +78,33 @@ def computeImage(self, image_pos):  # noqa
 
         return self._make_image(image_pos, offset=offset)
 
-    def computeKernelImage(self, image_pos):  # noqa
+    def computeKernelImage(self, image_pos, color=None):  # noqa
         """
         compute a centered kernel image appropriate for convolution
 
         Parameters
         ----------
         pos: geom.Point2D
             A point in the original image at which evaluate the kernel
+        color: afw_image.Color
+            A color, which is ignored
+        """
+
+        return self._doComputeKernelImage(
+            image_pos=image_pos,
+            color=color,
+        )
+
+    def _doComputeKernelImage(self, image_pos, color=None):  # noqa
```

Comment:
Is there DM code that accesses hidden methods?

Reply:
Yes :(

Reply:
Ouch. We should file a ticket if you have the details. This is bad form!

Reply:
agreed

Reply:
Its hard to track down because it happens in C++ and the names are different there

Reply:
Here is an official test where they do it

https://github.com/lsst/afw/blob/master/tests/test_psf_trampoline.py#L66

Reply:
If that is a must override it should be part of the public API